frontend/src/screens/WaitingRoom.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import Typography from "@material-ui/core/Typography";
+import EnterName from "../components/EnterName";
+import WaitingPlayers from "../components/WaitingPlayers";
+import Button from "@material-ui/core/Button";
+import { MyContext } from "../infra/MyContext.js";
+
+const MOCK_playerList = ["Elisha", "Lahav", "Aviad"];
+class WaitingRoom extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isUserEntered: false,
```

Comment:
do you really have to have this bool?

Reply:
Yes. 
When user enter the name its change and the page with the InputName is not longer needed